README.md

```diff
@@ -577,49 +571,6 @@ Every chapter interprets "awesome" for itself. As such, awesome projects include
 
 N/A
 
-## B
```

Comment:
Maybe we should also remove all the empty sections. This was the only entry for B so the section got removed and there are other empty sections iirc.